accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx

```diff
@@ -548,8 +548,8 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
                   <p style={{ marginTop: "0rem" }}>
                     To <strong>filter</strong> issues and focus only on
                     violations, items that need review or recommendations,
-                    select the issue type in the issue count region below the
-                    'Scan' button. Select the same issue type again to see all
+                    deselect the checkbox by the issue type in the issue count region below the
+                    'Scan' button to exclude the issue type from the results. Select the checkbox by the issue type to include the issue type in the results.
                     issue types.
```

Comment:
Does this line (553) 'issue types.' need to be deleted?

Reply:
yes, thanks.